server/middleware/renderer.js

```diff
@@ -7,6 +7,11 @@ const { App } = require('../../client/build/static/js')
 const path = require('path')
 const fs = require('fs')
 
+const { ServerStyleSheet, injectGlobal } = require('styled-components')
+const globalStyleRules = require('../../client/src/theme/globalStyleRules')
+
+injectGlobal`${globalStyleRules}`
```

Comment:
Should be in the exported function, unless there's a good reason to run it when requiring the file.

I'm always cautious with such side effects...